README.md

```diff
@@ -1,28 +1,85 @@
 # Introduction to Machine Learning for the Arts
 
+## OUTLINE
 
-## COURSE DESCRIPTION 
+### 1: Intro!
+* Session 1 (M 9/3): Introduction to Machine Learning
 
-An introductory course designed to provide students with hands-on experience developing creative coding projects with machine learning. The history, theory, and application of machine learning algorithms and related datasets are explored in a laboratory context of experimentation and discussion. Examples and exercises will be demonstrated in JavaScript using the p5.js, ml5.js, and TensorFlow.js libraries. In addition, students will learn to work with open source pre-trained models in the cloud using Runway. Principles of data collection and ethics are introduced. Weekly assignments, team and independent projects, and project reports are required. 
+### 2: Machine Learning Models 1
+* Session 1 (M 9/9): Image Classification
+* Session 2 (W 9/11): Transfer learning
+
+### 3: Machine Learning Models 2
+* Session 1 (M 9/16): Pre-trained models
+* Session 2 (W 9/18): Physical Interaction
+
+### 4: Training a Model
```

Comment:
Is this teachable machine / transfer learning? Or is this tf/ml5 neural network stuff?

Reply:
I am tentatively planning to get to teachable machine and transfer learning in week 2. This week would be introducing some other pre-trained models like UNet / PoseNet / TBD. The "physical interaction" module is an idea proposed by @ellennickles to encourage students to create physical output (turn on an LED based on a gesture). So no new ML concepts just trying to move the application off the screen. I would LOVE help from anyone for the pcomp side of this! 😬